addons/website_payment/models/res_config_settings.py

```diff
@@ -51,14 +51,18 @@ def action_activate_stripe(self):
         self.ensure_one()
         if not self.is_stripe_supported_country:
             return False
-        stripe = self.env.ref('payment.payment_provider_stripe')
+        stripe = self.env['payment.provider'].search(
+            [('company_id', '=', self.env.company.id), ('code', '=', 'stripe')]
+        )
```

Comment:
This will not find the provider by its `code` until the module is installed. Pay attention to the `env` that might need to be re-created to take the newly installed module into account.